docs/user-guide-sharing/citing-your-dandiset.md

```diff
@@ -0,0 +1,43 @@
+# Citing your own Dandiset
+
+After publishing your Dandiset, it's important to properly cite it in your research publications. This page provides guidance on how to cite your own Dandiset in your publications.
+
+## Why Cite Your Dandiset
+
+Citing your Dandiset is important because it:
+
+- Provides a direct link to the data used in your research
+- Facilitates reproducibility by allowing others to access the exact data you used
+- Helps track the impact of your data
+- Establishes your data as a citable research output, which can be tracked for impact metrics
+- Follows best practices for open science
+
+## How to Cite Your Dandiset
+
+When citing your Dandiset:
+
+1. **Use the DOI**: Each published version of your Dandiset receives a unique DOI (Digital Object Identifier). This DOI can be found on the Dandiset's landing page on the DANDI Archive website.
+
+2. **Use the "CITE AS" button**: The Dandiset's landing page provides an example formatted citation at the "CITE AS" button. This citation uses the DataCite citation style, which is a widely accepted standard for citing datasets.
+
+3. **Adapt to journal requirements**: You may need to adapt the citation format to match the citation style required by the journal you are submitting to.
+
+## Data Availability Statement
+
+Many journals require a Data Availability Statement in the manuscript. This statement should include the DANDI Archive RRID and the DOI of your Dandiset.
+
+Here is an example of a well-formatted Data Availability Statement:
+
+> The data that support the findings of this study are openly available on the DANDI Archive (RRID:SCR_017571) at [DOI of Dandiset] (citation of Dandiset).
+
+**Important**: A Data Availability Statement does not replace the need for a full citation in the manuscript's references section. Both elements serve different purposes and are typically required for comprehensive documentation of data sources.
+
+## Citing Both Dandiset and Publication
+
+If your Dandiset is associated with a publication:
+
+- Cite both the Dandiset and the publication in your references
+- In the Data Availability Statement, you can reference both the Dandiset and the publication
+- Link the Dandiset and publication in the "Related Resources" section of your Dandiset metadata
+
+This approach ensures that both your data and your analysis receive proper attribution.
```

Comment:
I might be misunderstanding, but perhaps a modified version of this text is better suited for the `User Guide: Using Data` section.

Reply:
ok, I removed this section and now we only have a section in the Using Data section